R/pkgs/covidcommon/R/config.R

```diff
@@ -0,0 +1,68 @@
+##'
+##'Map of configuration values, constructed via a call to `load_config`
+##'
+config <- NA
+
+##'
+##'Returns a map of configuration loaded from the config YAML
+##'@param fname Load configuration from fname (optional, otherwise loads from CONFIG_PATH env var)
+##'@example config$parameters_seir$gamma
+##'
+load_config <- function(fname) {
+  require(yaml)
+
+  if (missing(fname)) {
+    fname <- file.path(Sys.getenv("CONFIG_PATH"), "config.yml")
+  }
+  if (file.exists(fname)) {
+    return(yaml.load_file(fname))
+  } else {
+    return(NA)
+  }
+}
+
+##'
+##'Evaluates an expression, returning a numeric value
+##'@example as_evaled_expression(c("2+2", "9*9")) -> (4, 81)
+##'
+##'@param l the object (scalar or vector) to evaluate
+##'@return a float evaluation of the expression
+##'
+as_evaled_expression <- function(l) {
```

Comment:
any reason this always returns numeric? seems like there might be cases we would want to return another kind of thing (formula in particular seems plausible)

Reply:
also, this is super unsafe, assuming we don't care though